composer.json

```diff
@@ -10,7 +10,7 @@
         "issues": "https://github.com/llaville/php-reflect/issues"
     },
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^5.6|^7.0",
```

Comment:
The `doctrine/collections` `^1.4` version needs `php-5.6` version at least and let this package require `php-5.6` version at least.

Reply:
Hello. 
Thanks for your contribution, but perhaps you should have noticed that I'm working on a new master branch 5.0 that will drop support to PHP 5.